ctl/src/command.rs

```diff
@@ -247,92 +249,99 @@ pub fn upgrade<C>(mut channel: Channel<ConfigMessage,ConfigMessageAnswer>,
   ));
 
   match channel.read_message() {
-    None          => println!("the proxy didn't answer"),
+    None          => println!("Error: the proxy didn't list workers"),
```

Comment:
this case really means "the proxy did not answer", which is more information on the current state. Same for all other instances of this case afterwards

Reply:
I tried to find a balance between user friendliness and debuggability. Most of the time the those error messages tell little to the user, but having them be different should help in assisting finding the reason. If the proxy dies handling the n-th message it can be useful to know where to start the debugging from.

Though, if you think my reasoning above is not worth the changes, I can change it back to the "proxy didn't answer" message. Or to "proxy didn't answer [list workers message]"?